typespec-tests/src/test/java/azure/resourcemanager/resources/ArmResourceTest.java

```diff
@@ -66,6 +67,7 @@ public class ArmResourceTest {
     private ExtensionsResource extensionResource;
     private List<ExtensionsResource> extensionResources;
 
+    @Disabled
```

Comment:
to pass nightly build

change to the test is merged but not released

Reply:
Yeah, this should do.